### ISSUE TITLE
Build fails on macos due to unsupported `comm` flag `--nocheck-order`

### ISSUE THREAD

**Issue by olizilla** [ipfs/distributions#350]
The version of `comm` that ships with macos does not support the `--nocheck-order` flag which got added in https://github.com/ipfs/distributions/commit/19448fd1a5098fde6ce4f7c24d74a30884bba7e4

so macos users get

```console
** fs-repo-0-to-1 **
/Library/Developer/CommandLineTools/usr/bin/make -C dists/fs-repo-0-to-1
../../build-go.sh "fs-repo-0-to-1" "github.com/ipfs/fs-repo-migrations" "fs-repo-0-to-1" versions
comparing versions with /ipfs/QmUeJDaHiaETrpyqaydtp5Fgbkayz6qxkJ7n2Ddyi4pbpq/fs-repo-0-to-1/versions
comm: illegal option -- -
usage: comm [-123i] file1 file2
make[1]: *** [dist] Error 1
make: *** [fs-repo-0-to-1] Error 2
```

**Comment by olizilla** [comment]
This is so dull. This should probably be dockerized.

**Comment by aschmahmann** [comment]
@olizilla note that building in docker is almost certainly a good idea given #348 as well as #322

**Comment by lidel** [comment]
Closed by #352 and https://github.com/ipfs/distributions/pull/367 
TLDR: run `./dockerized make publish` or ./dockerized ./dist.sh`
